lib/okapi/endpoint-service.js

```diff
@@ -0,0 +1,37 @@
+const OkapiClient = require('./okapi-client');
+
+const parseJson = data => data.json();
+const stringifyJson = data => JSON.stringify(data, null, '  ');
+
+module.exports = class EndpointService {
+  constructor(okapi, tenant) {
+    this.client = new OkapiClient(okapi, tenant);
+  }
+
+  get(path) {
+    return this.client
+      .get(path)
+      .then(parseJson)
```

Comment:
If we're not doing anything with the JSON here, there really isn't a need parse it only to immediately stringify.  I realize that doesn't give the service much to do.  Maybe that's okay...  This is after all designed to target arbitrary endpoints, so by nature its a bit of a pass-through.  As much as I like having this layer, I wonder if this a valid case for the command to use the okapi-client directly.